src/Lightest.TestingService/DefaultServices/DefaultTestingService.cs

```diff
@@ -17,12 +18,14 @@ public class DefaultTestingService : ITestingService
         private readonly RelationalDbContext _context;
         private readonly IServerRepository _repository;
         private readonly ITransferServiceFactory _transferServiceFactory;
+        private readonly IUploadDataRepository _uploadDataRepository;
 
-        public DefaultTestingService(IServerRepository repository, RelationalDbContext context, ITransferServiceFactory transferServiceFactory)
+        public DefaultTestingService(IServerRepository repository, RelationalDbContext context, ITransferServiceFactory transferServiceFactory, IUploadDataRepository uploadDataRepository)
         {
             _context = context;
             _repository = repository;
             _transferServiceFactory = transferServiceFactory;
+            _uploadDataRepository = uploadDataRepository;
```

Comment:
Update the rest of code in this file to use `_uploadDataRepository ` and fix tests accordingly.